tests/dryrun/install_test.go

```diff
@@ -721,9 +737,88 @@ func TestInstall_HTTPProxy(t *testing.T) {
 		t.Fatalf("failed to create kube client: %v", err)
 	}
 
-	// TODO: CA
-	// assertConfigMapExists(t, kcli, "private-cas", "kotsadm")
-	// assertConfigMapExists(t, kcli, "kotsadm-private-cas", "embedded-cluster")
+	// --- validate installation object --- //
+	in, err := kubeutils.GetLatestInstallation(context.TODO(), kcli)
+	if err != nil {
+		t.Fatalf("failed to get latest installation: %v", err)
+	}
+
+	assert.Equal(t, hostCABundle, in.Spec.RuntimeConfig.HostCABundlePath)
+
+	// Verify some metrics were captured
+	assert.NotEmpty(t, dr.Metrics)
+}
+
+// Let's add a test for CA-only configuration without HTTP proxy settings
```

Comment:
hmm, why is this relevant? i wouldn't assume proxy settings to affect the ca bundle setting. i feel like it's permutation we shouldn't be testing and maintaining

Reply:
yeah, you're right. i'll remove it.